packages/core/src/index.ts

```diff
@@ -3,13 +3,13 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export {Component, ProviderMap} from './component';
-export * from './server';
+// Re-export public Core API coming from dependencies
+export * from '@loopback/context';
+// Export APIs
```

Comment:
Please add an empty line to visually delimit the block "Re-export..." from the block "Export APIs"